packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -446,6 +446,7 @@ class SizeTransition extends AnimatedWidget {
     this.axis = Axis.vertical,
     required Animation<double> sizeFactor,
     this.axisAlignment = 0.0,
+    this.crossAxisSizeFactor,
```

Comment:
we should probably assert this value is non-negative since `Align` cannot take a non-negative size factor.

Reply:
I copied the `Align`'s `assert` in [chore: Add assert and comment](https://github.com/flutter/flutter/pull/134659/commits/c4b5ca0055e9eb426256025a3bb31ffcb15994b2)